packages/fetchai/contracts/erc1155/contract.yaml

```diff
@@ -29,3 +29,4 @@ dependencies:
     version: <2.0.0,>=1.0.0
   open-aea-ledger-fetchai:
     version: <2.0.0,>=1.0.0
+contracts: []
```

Comment:
Please in the tests folder, add a stub contract which depends on another stub contract. Then write a test which shows that a) adding contract in aea project adds dependency too and b) running agent works too.